test/unit/lodash.spec.js

```diff
@@ -0,0 +1,690 @@
+            });
+            it('should include first value in the object', function () {
+                expect(_.includes.apply(_, [obj, predicate[0]])).to.eql(true);
+                expect(_.includes.apply(_, [obj, predicate[1]])).to.eql(false);
+            });
+        });
+        describe('includes2 function', function () {
+            var params = datasets.includes2();
+
+            it('should match types of passing elements', function () {
+                var element1 = params[0] instanceof Array;
+                var element2 = typeof params[1];
+
+                expect(element1).to.eql(true);
+                expect(element2).to.eql('string');
+                expect(params.length).to.eql(2);
+            });
+            it('should include \'stabli\' word in array of words', function () {
+                var array = params[0];
+                var predicate = params[1];
+
+                expect(predicate).to.have.length(6);
+                expect(array.length).to.eql(4);
+                expect(array).to.contain('antidisestablishmentarianism');
+                expect(array).to.not.contain('stabli');
+                array.forEach(function (element, index) {
+                    if (index !== 1) {
+                        expect(_.includes.apply(_, [element, predicate])).to.eql(true);
+                    }
+                    if (index == 1) {
+                        expect(element).to.have.length.above(5);
+                        expect(element).to.have.length.below(9);
+                        expect(_.includes.apply(_, [element, predicate])).to.eql(false);
+                    } else if (index == 2) {
+                        expect(element).to.have.length.above(10);
+                        expect(element).to.have.length.below(16);
+                    } else if (index == 3) {
+                        expect(element).to.have.length.above(6);
+                    }
+                });
+            });
+        });
+        describe('includes3 function', function () {
+            var array = [1, true, 3, 'wood', 4, 2, false, true, 'wodo'];
+            var params = datasets.includes3(array);
+            var predicate1 = params[1][0];
+            var predicate2 = params[1][1];
+            var predicate3 = params[1][2];
+
+            it('should match types of passing elements', function () {
+                var element = params[1] instanceof Array;
+
+                expect(element).to.eql(true);
+                expect(params[1]).to.have.length(3);
+                expect(params[1][0]).to.have.length(2);
+            });
+            it('should include relevant values as predicates(fromIndex)', function () {
+                var checkValues = _.concat([], predicate1, predicate2, predicate3);
+                expect(checkValues).to.include.members([5, 3, 1]);
+            });
+            it('should return true when predicates with fromIndex find the value', function () {
```

Comment:
Very confusing and long, but exercise is fine. It is good that you have tested fromIndex, but I don't think it is properly explained because there are quite a few meaningless examples.